packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableView.kt

```diff
@@ -217,13 +218,19 @@ class RNVisitableView(context: Context) : LinearLayout(context), SessionSubscrib
   }
 
   private fun showProgressView() {
-    turboView.addProgressView(inflate(context, R.layout.turbo_progress, null))
+    // Don't show the progress view if a screenshot is available
+    if (screenshotView.isVisible) return
```

Comment:
Hm, this differs from the [`turbo-android` implementation](https://github.com/hotwired/turbo-android/blob/b5ba0325e3a0ab5e87242a53fea5876d1322f4f5/turbo/src/main/kotlin/dev/hotwire/turbo/delegates/TurboWebFragmentDelegate.kt#L397-L399). Is it intentional?

Reply:
If we want this behavior, I feel like the guard clause is better fitted to have in the methods calling this. I find it weird that `showProgressView(..)` can sometimes be a no-op.

Reply:
Yes, the change is intentional. We no longer want to add native loader view to the `turboView`, instead we want to intercept all events responsible for showing a ActivityIindicator. 

This piece of code is also a copy-paste from [turbo-android](https://github.com/hotwired/turbo-android/blob/b5ba0325e3a0ab5e87242a53fea5876d1322f4f5/turbo/src/main/kotlin/dev/hotwire/turbo/views/TurboView.kt#L80-L88).

Reply:
Got it! Sounds reasonable and the way to go :-)

Reply:
Should we have the guard here for `isRefreshing` as well?

Reply:
A loading animation is triggered in a "refresh spinner":
![image](https://github.com/software-mansion-labs/react-native-turbo-demo/assets/44744462/f2fd504d-7e10-4d54-8fc0-871d9862b59d)